src/Tizen.Multimedia.Remoting/WebRTC/MediaCameraSource.cs

```diff
@@ -35,6 +35,39 @@ public sealed class MediaCameraSource : MediaSource
         /// <since_tizen> 9 </since_tizen>
         public MediaCameraSource() : base(MediaType.Video) {}
 
+        /// <summary>
+        /// Gets or sets the camera device id.
+        /// </summary>
+        /// <value>A value that specifies the camera device id.</value>
+        /// <exception cref="InvalidOperationException">MediaSource is not attached yet.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public uint CameraDeviceId
+        {
+            get
+            {
+                if (!SourceId.HasValue)
+                {
+                    throw new InvalidOperationException("MediaSource is not attached yet. Call AddSource() first.");
+                }
+
+                NativeWebRTC.GetCameraDeviceId(WebRtc.Handle, SourceId.Value, out uint deviceId).
+                    ThrowIfFailed("Failed to get camera devcie id");
```

Comment:
typo

Reply:
Did you mean that `camera devcie id` should be changed to `CameraDeviceId`?

Reply:
Fixed.